tests/test_table.py

```diff
@@ -25,19 +25,8 @@ def test_get_access_token(self):
     @testmode("workspace_access")
     def test_table(self):
         table_name = f"test_{uuid4()}"
-        number_of_entities = 5
         table.delete(table_name)  # remove cruft from previous failed tests
```

Comment:
I don't understand this test.  Is this testing the deletion of a non-existent table?

Reply:
These are updated in a separate PR.